package.json

```diff
@@ -29,14 +29,14 @@
     "exports": {
         ".": {
             "module": {
-              "types": "./tslib.d.ts",
-              "default": "./tslib.es6.js"
+              "types": "./tslib/modules/index.d.ts",
+              "default": "./tslib.es6.mjs"
             },
             "import": {
               "node": "./modules/index.js",
               "default": {
-                "types": "./tslib.d.ts",
-                "default": "./tslib.es6.js"
+                "types": "./modules/index.d.ts",
+                "default": "./tslib.es6.mjs"
```

Comment:
Do we have a test for a runtime whose conditions actually load this?

Reply:
vite and webpack-5 in the test directory should; possibly others